### DISCUSSION TITLE
Using FluxCD to bootstrap ArgoCD in new KinD Cluster.

### DISCUSSION THREAD

**Discussion by mondo192** [fluxcd/flux2#3814] [Q&A] [answered]
I am very impressed with FluxCD and use it for some personal learnings and projects. However, I would like to be able to use it to install ArgoCD to my cluster as well. 

I am following along to the documentation from [ArgoCD](https://argo-cd.readthedocs.io/en/stable/operator-manual/installation/#kustomize) to install it to a local development cluster via flux. However, it does not link to the correct version as I get a 404. I fixed the link to point to the right version and it still failed with: 

```bash
accumulation err='accumulating resources from 'github.com/argoproj/argo-cd/blob/v2.6.7/manifests/install.yaml': 
open /tmp/kustomization-490844070/clusters/kind/argocd/github.com/argoproj/argo-cd/blob/v2.6.7/manifests/install.yaml: 
no such file or directory'
```

I suspect I might be doing something wrong but I'm sure others have tried what I'm attempting in the past.

Would somebody from the community here be able to assist? Thank you

**Comment by kingdonb** [comment, marked as answer]
Howdy Eamonn! Have you seen Flamingo by chance?

🦩 This is the Flux Subsystem for Argo, it's intended to let you visualize your Flux resources in ArgoCD. The installation instructions cover adding an Argo to your Flux bootstraps.

https://github.com/flux-subsystem-argo/flamingo

Hope this helps! Let me know if you have trouble finding the right doc, it's a slightly different distro of Argo than upstream.

**Reply by kingdonb** [reply]
I haven't heard back, I'm going to mark this as "answered" – the Flamingo docs provide installation instructions so that Flux and Argo can coexist, and so that Argo can interface with the Flux (and Terraform Controller) resources. Let us know if you have trouble with this. Thanks for using Flux!
